viewer/packages/utilities/src/events/getPixelCoordinatesFromEvent.ts

```diff
@@ -0,0 +1,24 @@
+/*!
+ * Copyright 2021 Cognite AS
+ */
+
+import { Vector2 } from 'three';
+
+/**
+ * Determines clicked or touched coordinate as offset
+ * @param event     An PointerEvent or WheelEvent.
+ * @param target    HTML element to find coordinates relative to.
+ * @returns A Vector2 containing coordinates relative to the HTML element provided.
+ */
+export function getPixelCoordinatesFromEvent(event: PointerEvent | WheelEvent, target: HTMLElement): Vector2 {
+  const rect = target.getBoundingClientRect();
+  if (event instanceof PointerEvent) {
```

Comment:
I'm surprised that this works, I thought `PointerEvent` was an `interface`. But I see that they are doing some magic in the implementation file, so I guess it's fine :sweat_smile: